### PR TITLE
After chat setup outline 

### PR DESCRIPTION
**Pull request by vortexing**

### CODE REVIEW

File: 03-submission.Rmd [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+## Post-Submission Tips
+
+
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+## Learning Objectives
```
Comment (avahoffman) [inline 03-submission.Rmd:43]: Can we take out the rest of this? These are examples from the OTTR template

**Review by avahoffman — CHANGES_REQUESTED** [review]

File: 03-submission.Rmd (hunk 1 above)
Comment (carriewright11) [reply 03-submission.Rmd:43]: I just previously left them in for Amy

**Comment by carriewright11** [comment]
huh says we have a broken url for this: (updates can be made during the [Just-in-Time process](https://grants.nih.gov/grants/policy/nihgps/html5/section_2/2.5.1_just-in-time_procedures.htm)) but it seems fine in the previous PR

**Comment by carriewright11** [comment]
> huh says we have a broken url for this: (updates can be made during the [Just-in-Time process](https://grants.nih.gov/grants/policy/nihgps/html5/section_2/2.5.1_just-in-time_procedures.htm)) but it seems fine in the previous PR

nevermind... I think we have a duplicate report thing happening

**Comment by cansavvy** [comment]
> > huh says we have a broken url for this: (updates can be made during the [Just-in-Time process](https://grants.nih.gov/grants/policy/nihgps/html5/section_2/2.5.1_just-in-time_procedures.htm)) but it seems fine in the previous PR
> 
> nevermind... I think we have a duplicate report thing happening

Nah, this is related to: https://github.com/jhudsl/OTTR_Template/issues/590 which has actually been a long standing problem. You can add this url to ignore_urls.txt in the meantime if you don’t want the alert.

File: 03-submission.Rmd (hunk 1 above)
Comment (avahoffman) [reply 03-submission.Rmd:43]: gotcha 👍🏻

**Review by avahoffman — APPROVED** [review]

File: 03-submission.Rmd (hunk 1 above)
Comment (vortexing) [reply 03-submission.Rmd:43]: And I was hesitant to take them out in case it wasn't just for me. ;)

**Comment by carriewright11** [comment]
> > > huh says we have a broken url for this: (updates can be made during the [Just-in-Time process](https://grants.nih.gov/grants/policy/nihgps/html5/section_2/2.5.1_just-in-time_procedures.htm)) but it seems fine in the previous PR
> > 
> > 
> > nevermind... I think we have a duplicate report thing happening
> 
> Nah, this is related to: [jhudsl/OTTR_Template#590](https://github.com/jhudsl/OTTR_Template/issues/590) which has actually been a long standing problem. You can add this url to ignore_urls.txt in the meantime if you don’t want the alert.


hmm I did that be then got this:

Screen Shot 2022-10-21 at 3 00 34 PM

**Comment by carriewright11** [comment]
I think I will just add the remaining structure that we talked about if that's ok?

**Comment by cansavvy** [comment]
Okay. Thanks for letting me know. It’s apparently having a lot of trouble with this format of url but it should be made to be more flexible than this.

**Comment by carriewright11** [comment]
OK I added the structure that I thought we talked about - we can always undo the last commit if we don't like it. :) 

I added more chapters to make it easier for us to not overlap.

**Comment by avahoffman** [comment]
LGTM and we can always tweak it later :)

**Pull request merged**